### DISCUSSION TITLE
Full stack development in Python

### DISCUSSION THREAD

**Discussion by edzy16** [TheOdinProject/curriculum#24892] [Ideas]
I think python based web development is getting a lot of hype and Odin project needs a tutorial based on it.

**Comment by shubhxg** [comment]
Absolutely agreed.

**Comment by aScHEABL** [comment]
I second this idea. Loving TOP so far, after my college had an assignment that require me to write something in Flask, I have been using it consistently, and I would use it professionally in the future.
And Python in web development is much more relevant than RoR, at least in east Asia. Not everyone in this community is living in the north America.

**Comment by CouchofTomato** [comment]
The issue with writing a python track is someone has to maintain it.
